torch/csrc/profiler/kineto_shim.cpp

```diff
@@ -325,13 +335,18 @@ c10::DeviceType deviceTypeFromActivity(libkineto::ActivityType activity_type) {
     case libkineto::ActivityType::CUDA_PROFILER_RANGE:
     // TODO: T151322015
     case libkineto::ActivityType::MTIA_CCP_EVENTS:
-      return c10::DeviceType::CUDA;
+      TORCH_CHECK(
+          gpu_device != c10::DeviceType::CPU,
+          "Kineto GPU Activity Type enabled, but no available gpu device was found."
+          "Kineto allowed GPU device must be one of {CUDA, MTIA or XPU}");
+      return gpu_device;
```

Comment:
Since the activity type is MTIA CCP Event, may I know why we still need to check the gpu device is not GPU?